asapdiscovery-alchemy/asapdiscovery/alchemy/cli/alchemy.py

```diff
@@ -388,6 +426,10 @@ def status(network: str, errors: bool, with_traceback: bool, all_networks: bool)
     console = rich.get_console()
     print_header(console)
 
+    if network_key and any([all_networks, network]):
```

Comment:
To capture the full mutual exclusivity the following is possibly better
```
 if sum([bool(arg) for arg in [all_networks, network, network_key])] > 1:
     raise ValueError("Can only specify one of ")
```
Otherwise `--network` and `--all-networks` can coexist.